datawig/utils.py

```diff
@@ -395,3 +397,58 @@ def sample_cartesian(sets: List,
     assert width == 1  # at the end of this procedure, the leaf index set should have width 1.
 
     return out
+
+
+### Functionality for label shift detection. Experimental. ###
+
+def synthetic_label_shift_simple(N, label_proportions, error_proba, covariates=None) -> pd.DataFrame:
```

Comment:
This function is only used in tests I think, please move it to a test module, conftest.py could be a good place for this.